src/browser/atom-application.coffee

```diff
@@ -517,6 +517,8 @@ class AtomApplication
     return {pathToOpen} unless pathToOpen
     return {pathToOpen} if fs.existsSync(pathToOpen)
 
+    pathToOpen = pathToOpen.replace(/[:\s]*$/, '')
```

Comment:
> Super minor, but I'd go with `+` instead of `*` here just to make it a bit clearer what the intent is.

Yeah, that's definitely more clear -- using that in https://github.com/atom/atom/commit/6a7d5e69459cfd6dc14c2d4e25c97353f84c49c9 (I rebased that change).